src/sections/Home/styles.js

```diff
@@ -0,0 +1,17 @@
+import styled from 'styled-components';
+
+export const VideoHome = styled.video`
+    width: 100vw;
+    opacity: 0.6;
+    padding-top: ${({theme})=> theme.scale.base*6.5}px;
+    position: absolute;
+    z-index: -1;
```

Comment:
better than using z-index in every element that needs to be placed behind one in common, use z-index to place that component in common in front of every other one that needs to be placed behind (use `z-index:1000;` in `<Header/>`)